apps/prairielearn/src/pages/instructorAssessmentUploads/instructorAssessmentUploads.ts

```diff
@@ -23,12 +23,14 @@ router.get(
     res.locals.upload_job_sequences = await sqldb.queryRows(
```

Comment:
This can be converted to a local variable and parameter to the html file.

Reply:
Moved to local variable in [ad361b5](https://github.com/PrairieLearn/PrairieLearn/pull/9777/commits/ad361b501b5eaaa21891a8a7c389492448d3d51f).